content/en/database_monitoring/guide/managed_authentication.md

```diff
@@ -79,4 +76,5 @@ That's it!
 [5]: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html
 [6]: /database_monitoring/setup_postgres/rds/#grant-the-agent-access
 [7]: /database_monitoring/setup_postgres/aurora/#grant-the-agent-access
+[8]: /database_monitoring
```

Comment:
I'm not sure this is right, I want to redirect to the [getting_started](https://github.com/DataDog/documentation/pull/19431/files/8a33230c9a50d6f733bc8344f7ea69a5cf4bb869..6be091957fa189805ec1a3afb4f0c94471be90a1#diff-ccfec655adcc863c31452e5945bf9de533dd909d9401065357b2318db661cfd7R72) page

Reply:
@jmeunier28 That paragraph does have a link to the getting started page as well, just before this one. If as a user, I've already read past that one because I don't feel ready to configure DBM for whatever reason, I would expect to be linked to an explanation of DBM itself, since the linking sentence is explaining what DBM is. Does that make sense?

Reply:
ah sure 👍